packer_images/sweeper_logstash.conf.in

```diff
@@ -7,54 +7,165 @@ input {
         jdbc_validate_connection => true
         jdbc_driver_library => "/usr/share/logstash/postgresql-42.2.2.jar"
         jdbc_driver_class => "org.postgresql.Driver"
+        jdbc_default_timezone => "UTC"
```

Comment:
beware I think `created_date` and `last_modified` in CCD are not UTC. Those are local datetimes
I wouldn't change this now but tackle this as a separate ticket to limit risks